vitest.config.ts

```diff
@@ -9,7 +9,6 @@ export default defineConfig({
         autoUpdate: true,
       },
     },
-    dir: 'src',
```

Comment:
Vitest does some `cwd` magic when running test workers, so this didn't do what I thought it would..